core/scss/utilities/mixins/layout/_centered-grid-container.scss

```diff
@@ -0,0 +1,31 @@
+@charset 'UTF-8';
+
+//
+// Centered Grid Container
+//
+// Creates a responsive centered grid for all breakpoints
```

Comment:
Needs better documentation. Like, how when using you can only have one child element to what you have applied this to.